pkg/virt-controller/watch/workload-updater/workload-updater.go

```diff
@@ -289,11 +289,20 @@ func (c *WorkloadUpdateController) getUpdateData(kv *virtv1.KubeVirt) *updateDat
 		lookup[migration.Namespace+"/"+migration.Spec.VMIName] = true
 	}
 
+	liveMigrationFGEnabled := false
+	if kv.Spec.Configuration.DeveloperConfiguration != nil {
+		for _, fg := range kv.Spec.Configuration.DeveloperConfiguration.FeatureGates {
+			if fg == virtconfig.LiveMigrationGate {
+				liveMigrationFGEnabled = true
+			}
+		}
+	}
+
 	automatedMigrationAllowed := false
 	automatedShutdownAllowed := false
 
 	for _, method := range kv.Spec.WorkloadUpdateStrategy.WorkloadUpdateMethods {
-		if method == virtv1.WorkloadUpdateMethodLiveMigrate {
+		if method == virtv1.WorkloadUpdateMethodLiveMigrate && liveMigrationFGEnabled {
```

Comment:
Would it make sense to have an event in case `workloadUpdateMethods` has the `WorkloadUpdateMethodLiveMigrate` but the feature gate is disabled?